submissions/m-ruslan/html-css-popup/index.html

```diff
@@ -0,0 +1,194 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="app.css" />
+    <link rel="icon" type="image/png" href="favicon.png" />
+    <title>Popup</title>
+  </head>
+  <body>
+    <nav>
+      <div class="nav-container">
```

Comment:
Some unofficial rule of creating header navigation is to use such structure `header>ul>li>a`  ([more info](https://developer.mozilla.org/ru/docs/Web/HTML/Element/nav))